eduvpn/main.py

```diff
@@ -55,9 +52,11 @@ def parse_args():
 
 
 def init(lets_connect):
+    #type: (bool) -> EduVpnApp
+    """Initialise the main app."""
```

Comment:
not sure if adding this comment to the `init()` function in the `main` file has added value.

Reply:
Removed the comment.